src/main/java/com/jfrog/ide/idea/ui/issues/IssuesTab.java

```diff
@@ -131,7 +121,7 @@ private JComponent createComponentsDetailsView(boolean supported) {
         JLabel title = new JBLabel(" Component Details");
         title.setFont(title.getFont().deriveFont(TITLE_FONT_SIZE));
 
-        issuesDetailsPanel = new JBPanel(new BorderLayout()).withBackground(UIUtil.getTableBackground());
+        issuesDetailsPanel = new JBPanel<>(new BorderLayout()).withBackground(UIUtil.getTableBackground());
```

Comment:
To get the whole line selected:
```suggestion
        issuesDetailsPanel = new JBPanel<>(new GridLayout()).withBackground(UIUtil.getTableBackground());
```

Reply:
Applied your suggestion in line 88.